internal/tools/update_specification_version.sh

```diff
@@ -12,13 +12,16 @@ LATEST_SPECIFICATION_VERSION="v1.26.0"
 # The specific pattern we look for when replacing URLs
 SPECIFICATION_URL_PREFIX="https://github.com/open-telemetry/opentelemetry-specification/tree/"
 SPECIFICATION_BLOB_URL_PREFIX="https://github.com/open-telemetry/opentelemetry-specification/blob/"
+# The specific pattern we look for updating the Badge with the spec version
+SPECIFICATION_BADGE_PREFIX="https://img.shields.io/badge/Specification_version-"
```

Comment:
```suggestion
SPECIFICATION_BADGE_PREFIX="https://img.shields.io/badge/OTel_specification_version-"
```
If https://github.com/open-telemetry/semantic-conventions/pull/558/files#r1403546995 is accepted.